Pocket.xcodeproj/project.pbxproj

```diff
@@ -58,6 +58,8 @@
 		16FCADBF26CB04AC00906C03 /* ArchiveAnItemTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 16FCADBE26CB04AC00906C03 /* ArchiveAnItemTests.swift */; };
```

Comment:
Lots of changes to the Xcode project. Most have to do with temporarily adjusting signing and just adding some experimental stuff around app groups.